libnavigation-base/src/test/java/com/mapbox/navigation/base/options/IncidentsOptionsTest.kt

```diff
@@ -0,0 +1,18 @@
+package com.mapbox.navigation.base.options
+
+import com.mapbox.navigation.testing.BuilderTest
+import org.junit.Test
+
+class IncidentsOptionsTest : BuilderTest<IncidentsOptions, IncidentsOptions.Builder>() {
+
+    override fun getImplementationClass() = IncidentsOptions::class
+
+    override fun getFilledUpBuilder() = IncidentsOptions.Builder()
+        .graph("graph")
+        .apiUrl("apiUrl")
```

Comment:
Can we use real values to know / understand / document better how these APIs work?

Reply:
it's just a Builder test, mock values can be used